services/user/PsiSpaceSys/ui/vite.config-default-profile.ts

```diff
@@ -22,7 +22,9 @@ const psibase = (appletContract: string, isServing?: boolean) => {
     if (isServing) {
         buildAliases.push({
             find: "@psibase/common-lib",
-            replacement: path.resolve("../../CommonSys/common-lib/src"),
+            replacement: path.resolve(
+                "../../CommonSys/common/packages/rpc/src"
+            ),
         });
```

Comment:
None of the other apps have this file, right? Did you ever figure out why this default-profile version exists? If not, I'd strip it out.

Reply:
That's in the territory of app cleanup. And I think that should happen when we work on getting these apps working again.

I'd like to keep this PR as narrowly focused on the RPC/keyConversions stuff as possible.